.NET/Microsoft.Recognizers.Text.DataDrivenTests/TestBase.cs

```diff
@@ -91,7 +91,17 @@ public void TestNumber()
 
                 Assert.AreEqual(expected.TypeName, actual.TypeName, GetMessage(TestSpec));
                 Assert.AreEqual(expected.Text, actual.Text, GetMessage(TestSpec));
-                Assert.AreEqual(expected.Resolution["value"], actual.Resolution["value"], GetMessage(TestSpec));
+
+                if (TestContext.TestName == "NumberRangeModel")
```

Comment:
I suggest that create a new method TestNumberRangeModel instead to mix them together.